community/SDK-GOVERNANCE.md

```diff
@@ -108,7 +108,7 @@ enhancements.
 
 ### Handover to a new maintainer/group of maintainers
 
-If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, the community MAY
+If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, the community may
```

Comment:
...the community may **decide** to **hand over** the project...

Reply:
next PR